src/sparselm/model_selection.py

```diff
@@ -264,6 +268,9 @@ def fit(self, X, y=None, *, groups=None, **fit_params):
         X, y, groups = indexable(X, y, groups)
         fit_params = _check_fit_params(X, fit_params)
 
+        # Overwrite default CV splitter to RepeatedKFold and force shuffling before split.
+        if self.cv is None:
```

Comment:
can you have a look a the check raised by codacy here.

Reply:
I removed setting self.cv to RepeatedKFold as default because that might not be very conventional. The users should be responsible for setting it when they initialize model selection.